Спринт 14.

Kris-found opened the pull request:
Обновлен экран "Поиска":
- Добавлен ProgressBar при загрузке до получения ответа от сервера
- Отправка запроса происходит автоматически
- Выполнение поискового запроса происходит через 2 сек после окончания пользовательского ввода
- Нажатие на элемента трека происходит не чаще одного раза в секунду Обновлен экран "Аудиоплеер"
- Добавлен MediaPlayer
- Добавлено воспроизведение 30-сек отрывка трека
- Вопросизведение начинается с нажатия на кнопку "play", остановка на кнопку "pause"
- Добвавлен таймер отображения текущего прогресса воспроизведения отрывка трека

Kris-found committed:
Спринт 14.
Обновлен экран "Поиска":
- Добавлен ProgressBar при загрузке до получения ответа от сервера
- Отправка запроса происходит автоматически
- Выполнение поискового запроса происходит через 2 сек после окончания пользовательского ввода
- Нажатие на элемента трека происходит не чаще одного раза в секунду
Обновлен экран "Аудиоплеер"
- Добавлен MediaPlayer
- Добавлено воспроизведение 30-сек отрывка трека
- Вопросизведение начинается с нажатия на кнопку "play", остановка на кнопку "pause"
- Добвавлен таймер отображения текущего прогресса воспроизведения отрывка трека

IliaAnK commented on app/src/main/java/com/practicum/playlistmaker/SearchActivity.kt:
```diff
@@ -35,20 +50,22 @@ class SearchActivity : AppCompatActivity() {
         .build()
 
     private val iTunesService = retrofit.create(ITunesSearchAPI::class.java)
```
Рекомендация:
Создание всего ретрофит клиента можно вынести в отдельный класс/файл

Kris-found committed:
Спринт 15.
Проект разделен на слои Data, Domain и Presentation, существующая логика перенесена в соответствующие слои
